translations/copyq_it.ts

```diff
@@ -271,176 +271,176 @@ Potete impostare il comando nelle preferenze.</translation>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="106"/>
         <source>Copy previous item</source>
-        <translation type="unfinished"></translation>
+        <translation>Copia elemento precedente</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="108"/>
         <source>Paste clipboard as plain text</source>
-        <translation type="unfinished"></translation>
+        <translation>Incolla Appunto come testo semplice</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="110"/>
         <source>Disable clipboard storing</source>
-        <translation type="unfinished"></translation>
+        <translation>Disabilita cronologia Appunti</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="112"/>
         <source>Enable clipboard storing</source>
-        <translation type="unfinished"></translation>
+        <translation>Abilita cronologia Appunti</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="114"/>
         <source>Paste and copy next</source>
-        <translation type="unfinished"></translation>
+        <translation>Incolla e copia successivo</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="116"/>
         <source>Paste and copy previous</source>
-        <translation type="unfinished"></translation>
+        <translation>Incolla e copia precedente</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="137"/>
         <source>New command</source>
-        <translation type="unfinished"></translation>
+        <translation>Nuovo commando</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="162"/>
         <source>Ignore items with no or single character</source>
-        <translation type="unfinished"></translation>
+        <translation>Ignora elementi con nessuno o singolo carattere</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="169"/>
         <source>Open in &amp;Browser</source>
-        <translation type="unfinished"></translation>
+        <translation>Apri nel &amp;browser</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="177"/>
         <source>Paste as Plain Text</source>
-        <translation type="unfinished"></translation>
+        <translation>Incolla come testo semplice</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="183"/>
         <source>Shift+Return</source>
-        <translation type="unfinished"></translation>
+        <translation>Maiusc+INVIO</translation>
```

Comment:
Same as above, this is recognized only as `Shift+Invio` (though letter-case doesn't matter).